lib/matplotlib/colors.py

```diff
@@ -2888,6 +2898,14 @@ def inverse(self, value):
                 raise ValueError("Not invertible until scaled")
             if self.vmin > self.vmax:
                 raise ValueError("vmin must be less or equal to vmax")
+
+            if norm_before_trf:
+                value, is_scalar = self.process_value(value)
+                value = self._trf.inverted().transform(value).reshape(np.shape(value))
+                value *= (self.vmax - self.vmin)
+                value += self.vmin
+                return value[0] if is_scalar else value
```

Comment:
Use `rescaled` to denote the **rescaled** value.

```suggestion
            if norm_before_trf:
                value = self._trf.inverted().transform(value).reshape(np.shape(value))
                rescaled = value * (self.vmax - self.vmin)
                rescaled += self.vmin
                return rescaled[0] if is_scalar else rescaled
```